### PR TITLE
Healing hands mutation

### PR DESCRIPTION
**Pull request by ghost**
basically touch people to heal 5 brute+burn damage. chance to also heal tox or oxy damage. doesn't work on androids/synths/borgs. 7.5 second cooldown.

it works but it needs sprites

:cl: More Robust Than You
add: Healing hands mutation
/:cl:

### CODE REVIEW

**Comment by AnonymousNow** [comment]
thumbs up

# I have been waiting for this for a very long time.

**Comment by Supermichael777** [comment]
does it work on the user?

File: code/datums/mutations.dm [hunk 1]
```diff
@@ -646,6 +646,60 @@ GLOBAL_LIST_EMPTY(mutations_list)
...
 		owner.LaserEyes(target)
 
+/datum/mutation/human/healing_hands
+	name = "Healing Hands"
+	quality = POSITIVE
+	get_chance = 27.5
+	lowest_value = 256 * 12
+	text_gain_indication = "<span class='notice'>You feel a comforting buzzing in your hands.</span>"
+	time_coeff = 5
+	var/next_heal_time = 0
```
Comment (Cruix) [inline code/datums/mutations.dm]: Mutation datums are singletons, so everyone who has the mutation uses the same datum instance. If you do it this way, everyone who has the mutation will share the same cooldown. If you want everyone to have their own cooldown, you will have to store it in a var on the mob or in an associative list on the datum.

**Comment by Repukan** [comment]
Make it not work on self use.

**Comment by Anonmare** [comment]
Should make your punches heal people.

Make the security officer go ballistic as you beat *from* an inch of his life

File: code/datums/mutations.dm [hunk 2]
```diff
@@ -646,6 +646,60 @@ GLOBAL_LIST_EMPTY(mutations_list)
...
+	if(iscarbon(target))
+		var/mob/living/carbon/human/H = target
+		if(H.dna && H.dna.species && (istype(H.dna.species, /datum/species/android) || istype(H.dna.species, /datum/species/synth)))
+			to_chat(owner, "<span class='warning'>You try to heal [H], but nothing happens.</span>")
+			return
+		H.heal_overall_damage(5,5)
+		if(prob(25))
+			H.adjustOxyLoss(-5)
+		if(prob(7.5))
+			H.adjustToxLoss(-2.5)
```
Comment (ExcessiveUseOfCobblestone) [inline code/datums/mutations.dm]: Please add this to the OP.

**Comment by AnonymousNow** [comment]
I'm of the opinion that taking away your ability to harm things through punches would be grand (especially if you're a golem or hulk), but making it incapable of healing yourself - along with its cooldown - is a little meh.

Maybe give it a longer cooldown if you use it on yourself? Is that possible?

**Comment by Repukan** [comment]
No job deserved free self healing, cool down or not.

**Comment by KorPhaeron** [comment]
We don't need more sources of unlimited healing or more genetics powers until we fix genetics being a complete free for all once you research something once

**Comment by Mercenaryblue** [comment]
We could have the ability in the code, but not easily accessible.

**Comment by AnonymousNow** [comment]
Jeeeeeeeeeee-sus, that tanked fast. Maybe I should've claimed to hate it instead.

Genetics definitely needs a rework either way, but when it's done, this needs to be in.

**Comment by jughu** [comment]
Imagine genetics having a power that profits medbay and heals people then validhunt powers

**Pull request closed without merging**